### PR TITLE
path-fix release notes

### PR DESCRIPTION
**Pull request by rdodev**
Quick release notes for patch-fix.

### CODE REVIEW

**Comment by rdodev** [comment]
@jwitz when you get a chance 👀

File: enterprise/v0.25/10_resources/01_release-notes.md [hunk 1]
```diff
@@ -12,6 +12,15 @@ If you're looking to upgrade to Astronomer v0.25, refer to [Upgrade to Astronome
 
 We're committed to testing all Astronomer Enterprise versions for scale, reliability and security on Amazon EKS, Google GKE and Azure AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](https://support.astronomer.io).
 
+## v0.25.7
+
+Release date: Augusst 9, 2021
+
+* Patch release to address issues related to hard-delete.
```
Comment (jwitz) [inline enterprise/v0.25/10_resources/01_release-notes.md:19]:
```suggestion
### Minor changes

* Fixed issues related to hard deleting Deployments.
```

File: enterprise/v0.25/10_resources/01_release-notes.md [hunk 2]
```diff
@@ -12,6 +12,15 @@ If you're looking to upgrade to Astronomer v0.25, refer to [Upgrade to Astronome
...
+* Removes support for Kubernetes versions older than 1.16.
+* Adds support for Kubernetes 1.21.
+
+
```
Comment (jwitz) [inline enterprise/v0.25/10_resources/01_release-notes.md:23]:
```suggestion
```

File: enterprise/v0.25/10_resources/01_release-notes.md [hunk 3]
```diff
@@ -12,6 +12,15 @@ If you're looking to upgrade to Astronomer v0.25, refer to [Upgrade to Astronome
 
 We're committed to testing all Astronomer Enterprise versions for scale, reliability and security on Amazon EKS, Google GKE and Azure AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](https://support.astronomer.io).
 
+## v0.25.7
+
+Release date: Augusst 9, 2021
+
+* Patch release to address issues related to hard-delete.
+* Removes support for Kubernetes versions older than 1.16.
```
Comment (jwitz) [inline enterprise/v0.25/10_resources/01_release-notes.md:20]:
```suggestion
* Removed support for Kubernetes versions older than 1.16.
```

File: enterprise/v0.25/10_resources/01_release-notes.md [hunk 4]
```diff
@@ -12,6 +12,15 @@ If you're looking to upgrade to Astronomer v0.25, refer to [Upgrade to Astronome
...
+* Adds support for Kubernetes 1.21.
```
Comment (jwitz) [inline enterprise/v0.25/10_resources/01_release-notes.md:21]:
```suggestion
* Added support for Kubernetes 1.21.
```

**Review by jwitz — APPROVED** [review]
Just made changes to the version compatibility matrix and some wording in the release notes. This should be good to go now!

**Pull request merged**